certbot-dns-google/certbot_dns_google/dns_google.py

```diff
@@ -154,14 +161,18 @@ def del_txt_record(self, domain, record_name, record_content, record_ttl):
             logger.warn('Error finding zone. Skipping cleanup.')
             return
 
+        record_contents = self.get_existing_txt_rrset(zone_id, record_name)
+        if not record_contents:
+            return
+
         data = {
             "kind": "dns#change",
             "deletions": [
                 {
                     "kind": "dns#resourceRecordSet",
                     "type": "TXT",
                     "name": record_name + ".",
-                    "rrdatas": [record_content, ],
+                    "rrdatas": record_contents,
```

Comment:
nit: Similar to my above comment, this deletes all records rather than the one that Certbot is asking it to. I believe this could be fixed fairly easily by adding `additions` containing `record_contents` with `record_content` removed.

Reply:
This is now done